engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/client/ConfluentSchemaRegistryClientFactory.scala

```diff
@@ -0,0 +1,8 @@
+package pl.touk.nussknacker.engine.avro.schemaregistry.confluent.client
+
+import pl.touk.nussknacker.engine.avro.schemaregistry.SchemaRegistryClientFactory
+import pl.touk.nussknacker.engine.kafka.KafkaConfig
+
+trait ConfluentSchemaRegistryClientFactory extends SchemaRegistryClientFactory {
```

Comment:
I thinkSchemaRegistryClientFactory is not really needed? It's used only in context of implementation like Confluent?

Reply:
Right now we don't need it - I've removed it.

Reply:
You're right - removed.